packages/cactus-plugin-ledger-connector-fabric/README.md

```diff
@@ -1,75 +1,118 @@
-# `@hyperledger/cactus-plugin-ledger-connector-fabric`
+# `@hyperledger/cactus-plugin-ledger-connector-fabric` <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+
+- [1. Usage](#1-usage)
+  - [1.1. Installation](#11-installation)
+  - [1.2. Using as a Library](#12-using-as-a-library)
+  - [1.3. Using Via The API Client](#13-using-via-the-api-client)
+  - [1.4. Signing Credentials with Hashicorp Vault](#14-signing-credentials-with-hashicorp-vault)
+    - [1.4.1. Identity Providers](#141-identity-providers)
+    - [1.4.2. Setting up a WS-X.509 provider](#142-setting-up-a-ws-x509-provider)
+    - [1.4.3. Building the ws-identity docker image](#143-building-the-ws-identity-docker-image)
+- [2. Architecture](#2-architecture)
+  - [2.1. run-transaction-endpoint](#21-run-transaction-endpoint)
+- [3. Containerization](#3-containerization)
+  - [3.1. Building/running the container image locally](#31-buildingrunning-the-container-image-locally)
+  - [3.2. Running the container](#32-running-the-container)
+  - [3.3. Testing API calls with the container](#33-testing-api-calls-with-the-container)
+- [4. Prometheus Exporter](#4-prometheus-exporter)
+  - [4.1. Usage Prometheus](#41-usage-prometheus)
+  - [4.2. Prometheus Integration](#42-prometheus-integration)
+  - [4.3. Helper code](#43-helper-code)
+    - [4.3.1. response.type.ts](#431-responsetypets)
+    - [4.3.2. data-fetcher.ts](#432-data-fetcherts)
+    - [4.3.3. metrics.ts](#433-metricsts)
+- [5. Contributing](#5-contributing)
+- [6. License](#6-license)
+- [7. Acknowledgments](#7-acknowledgments)
+
+## 1. Usage
+
+This plugin provides a way to interact with Fabric networks. 
+Using this one can perform:
+* Deploy smart contracts (chaincode).
+* Execute transactions on the ledger.
+* Invoke chaincode functions.
+
+The above functionality can either be accessed by importing the plugin directly as a library (embedding) or by hosting it as a REST API through the [Cactus API server](https://www.npmjs.com/package/@hyperledger/cactus-cmd-api-server)
+
+We also publish the [Cactus API server as a container image](https://github.com/hyperledger/cactus/pkgs/container/cactus-cmd-api-server) to the GitHub Container Registry that you can run easily with a one liner.
```

Comment:
This reference URL points to the cactus-cmd-api-server, but that github link doesn't hold any docs around its usage or references (will be good, if the [catus-cmd-api-server container image](https://github.com/hyperledger/cactus/pkgs/container/cactus-cmd-api-server) can have reference to these URL's

https://github.com/hyperledger/cactus/tree/main/packages/cactus-cmd-api-server
https://github.com/hyperledger/cactus/blob/main/BUILD.md

Reply:
@jagpreetsinghsasan This is link shows the fully qualified image name and the available versions which are IMO some of the most important information about the usage of the container image. The other link above it points to the npm package which shows the same readme that you linked.